src/libpriv/rpmostree-rpm-util.c

```diff
@@ -1057,3 +1057,25 @@ rpmostree_fcap_to_xattr_variant (const char *fcap)
                                                    vfsbytes, FALSE));
   return g_variant_ref_sink (g_variant_builder_end (&builder));
 }
+
+gboolean
+rpmostree_get_pkg_chksum_repr (DnfPackage *pkg,
+                               char      **out_chksum_repr,
+                               GError    **error)
+{
+  int chksum_type;
+  g_autofree char *chksum = NULL;
+  const unsigned char *chksum_raw = NULL;
+
+  chksum_raw = dnf_package_get_chksum (pkg, &chksum_type);
```

Comment:
The thing that feels weird about this is that in repodata, there can be an array of possible checksums; at least, I believe so.  `createrepo_c` has:

>    -s --checksum <checksum_type>
>      Choose the checksum type used in repomd.xml and for packages in the metadata. The default is now sha256.

If at some point people start transitioning by adding a new hash (say something SHA-3 based or whatever), the pkgcache will re-import the RPMs.  Which is mostly OK except...hm, it looks like the libsolv repomd XML parser doesn't have any logic for handling multiple checksum entries, it's last one wins?   Which would make things a bit fragile.  And oh man, that's enough reading the libsolv code for today.

In practice I'm OK deferring the pain here until for some reason we try to do a checksum transition in the ecosystem.

Maybe just add a comment like: `/* In practice this will be sha256, see createrepo_c -s */`